log-analyzer/features/org.wso2.carbon.la.apim.feature/src/main/resources/gadgets/LogAPIMArtifactDeleted/js/VizGrammar.js

```diff
@@ -0,0 +1,2300 @@
+    };
+
+    var yScale = {
+        "name": "y",
+        "type": this.metadata.types[config.y],
+        "range": "height",
+        "zero": "false",
+        "domain": {"data":  config.title, "field": this.metadata.names[config.y]}
+    };
+
+    var scales =  [xScale, yScale];
+
+    if (config.color != -1) {
+
+        if (config.colorDomain == -1) {
+            config.colorDomain = {"data":  config.title, "field": this.metadata.names[config.color]};
+        }
+
+        var colorScale = {
+            "name": "color",
+            "type": "ordinal",
+            "domain": config.colorDomain,
+            "range": config.colorScale
+        };
+        scales.push(colorScale);
+
+        var legendTitle = "Legend";
+
+        if (config.title != "table") {
+            legendTitle = config.title;
+        }
+
+        var legends = [
+            {
+                "fill": "color",
+                "title": "Legend",
+                "offset": 0,
+                "properties": {
+                    "symbols": {
+                        "stroke": {"value": "transparent"}
+                    },
+                    "title": {
+                        "fill": {"value": config.legendTitleColor},
+                        "fontSize": {"value": config.legendTitleFontSize}
+                    },
+                    "labels": {
+                        "fill": {"value": config.legendTextColor},
+                        "fontSize": {"value": config.ledgendTextFontSize}
+                    }
+                }
+            }
+        ];
+
+        this.spec.legends = legends;
+    }
+
+
+    var axes =  getXYAxes(config, "x", "x", "y", "y");
+
+    marks.push(getAreaMark(config, this.metadata));
+    config.fillOpacity  = 0;
+    config.markSize = 1000;
+    marks.push(getSymbolMark(config, this.metadata));
+
+    this.spec.width = config.width;
+    this.spec.height = config.height;
+    this.spec.axes = axes;
+    this.spec.data = dataTable;
+    this.spec.scales = scales;
+    this.spec.padding = config.padding;
+    this.spec.marks = marks;
+};
+
+area.prototype.draw = function(div, callbacks) {
+
+    var viewUpdateFunction = (function(chart) {
+        if(this.config.tooltip.enabled){
+            createTooltip(div);
+            this.view = chart({el:div}).renderer(this.config.renderer).update();
+            bindTooltip(div,this.view,this.config,this.metadata);
+        } else {
+            this.view = chart({el:div}).renderer(this.config.renderer).update();
+        }
+
+        if (callbacks != null) {
+            for (var i = 0; i<callbacks.length; i++) {
+                this.view.on(callbacks[i].type, callbacks[i].callback);
+            }
+        }
+
+    }).bind(this);
+
+    if(this.config.maxLength != -1){
+        var dataset = this.spec.data[0].values;
+        var maxValue = this.config.maxLength;
+        if(dataset.length >= this.config.maxLength){
```

Comment:
Hi Chandana,
We have replace the Vizgrammer.js with Vizgrammer.min.js and relevant commits are in

https://github.com/wso2/shared-analytics/pull/126/files#diff-6ac780d61b2b6d470c8ff04bdeed3bcdR52

Thanks,
SajithD